File: app/src/main/java/com/main/climbingdiary/common/AlertManager.java
package com.main.climbingdiary.common;

import android.content.Context;

import com.main.climbingdiary.models.Alert;

import cn.pedant.SweetAlert.SweetAlertDialog;

public class AlertManager {
    public static void setErrorAlert(Context context) {
        new SweetAlertDialog(context, SweetAlertDialog.ERROR_TYPE)
                .setTitleText("Oops...")
                .setContentText("Es ist ein Fehler aufgetreten.")
                .show();
    }

    public void setAlertWithoutContent(Context context, Alert alert) {
        new SweetAlertDialog(context, alert.getDialogType())
                .setTitleText(alert.getTitle())
                .show();
    }
}

File: app/src/main/java/com/main/climbingdiary/common/AlertManager.kt
package com.main.climbingdiary.common

import android.content.Context
import cn.pedant.SweetAlert.SweetAlertDialog
import com.main.climbingdiary.models.Alert

object AlertManager {

    fun setErrorAlert(context:Context){
        SweetAlertDialog(context, SweetAlertDialog.ERROR_TYPE)
            .setTitleText("Oops...")
            .setContentText("Es ist ein Fehler aufgetreten.")
            .show()
    }

    fun setAlert(context: Context, alert:Alert){
        SweetAlertDialog(context, alert.dialogType!!)
            .setTitleText(alert.title)
            .show()
    }
}